<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureEditModesDialog</class>
 <widget class="QDialog" name="ConfigureEditModesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>XSLT mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowXSLTPanel">
     <property name="text">
      <string>Show the &quot;Switch to XSLT&quot; panel when opening files.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAutomaticallySwitchToXSLTMode">
     <property name="text">
      <string>Automatically switch to XSLT edit mode when opening an XSLT file.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SCXML mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowSCXMLPanel">
     <property name="text">
      <string>Show the &quot;Switch to SCXML&quot; panel when opening files.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAutomaticallySwitchToSCXMLMode">
     <property name="text">
      <string>Automatically switch to SCXML edit mode when opening a SCXML file.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
